geoportal/geoportailv3_geoportal/static-ngeo/js/feedbackcrues/FeedbackcruesController.js

```diff
@@ -95,30 +98,35 @@ const exports = function($scope, $http, appNotify, appUserManager,
    */
   this.url = '';
 
-  $scope.$watch(
+  this.scope_.$watch(() => this['sidebarActive'], newVal => {
+    if (newVal) {
+      this['active'] = false;
+    }
+  });
+};
+
+exports.prototype.$onInit = function() {
+  this.map_ = this['map'];
+
+  this.scope_.$watch(
```

Comment:
same that before for $watch and angular version

Reply:
see above :-)